### PR TITLE
Integration issues

### PR DESCRIPTION
**Pull request by foxeyes**

### CODE REVIEW

File: blocks/DataOutput/DataOutput.js [hunk 1]
```diff
@@ -22,63 +22,71 @@ export class DataOutput extends UploaderBlock {
...
   initCallback() {
     super.initCallback();
-    this.sub('output', (data) => {
-      if (!data) {
-        return;
-      }
+    this.sub(
+      'output',
+      (data) => {
+        if (!data) {
```
Comment (FilippovAM) [inline blocks/DataOutput/DataOutput.js:28]: May be?
```
!data || data?.length === 0
```

File: blocks/DropArea/DropArea.js [hunk 2]
```diff
@@ -17,13 +18,28 @@ export class DropArea extends UploaderBlock {
...
         files.forEach((/** @type {File} */ file) => {
-          this.uploadCollection.add({
-            file,
-            isImage: file.type.includes('image'),
-            mimeType: file.type,
-            fileName: file.name,
-            fileSize: file.size,
-          });
+          let hasValidationErrors = false;
+          let isImage = file.type.includes('image');
```
Comment (FilippovAM) [inline blocks/DropArea/DropArea.js:22]: minor
let isImage => const isImage

**Review by FilippovAM — APPROVED** [review]

File: blocks/DataOutput/DataOutput.js (hunk 1 above)
Comment (foxeyes) [reply blocks/DataOutput/DataOutput.js:28]: Empty array is also valid value for output.

File: blocks/DropArea/DropArea.js (hunk 2 above)
Comment (foxeyes) [reply blocks/DropArea/DropArea.js:22]: We don't use "prefer const" convention for some reason.

**Pull request merged**